src/mycha_installer.coffee

```diff
@@ -19,10 +19,13 @@ class MychaInstaller
 
 
   _installDependencies: (done) =>
-    @_writeToStdout colors.bold('npm install --save-dev chai sinon sinon-chai'), lineBefore: yes
-    npm.load 'save-dev': yes, (err) =>
-      if err then return done err
-      npm.commands.install @cwd, ['chai', 'sinon', 'sinon-chai'], done
+    cmd = 'npm'
+    args = ['install', '--save-dev', 'chai', 'sinon', 'sinon-chai']
+    cmdString = [cmd].concat(args).join ' '
+    @_writeToStdout colors.bold(cmdString), lineBefore: yes
```

Comment:
Maybe get rid of the `cmdString` temp variable here? It doesn't add readability in my opinion.

```coffeescript
@_writeToStdout colors.bold([cmd].concat(args).join ' '), lineBefore: yes